config/other.go

```diff
@@ -215,8 +215,8 @@ func setDefaults(conf *Config, bus awlevent.Bus) {
 	if !conf.Update.TrayAutoCheckEnabled && conf.Update.TrayAutoCheckInterval == "" {
 		conf.Update.TrayAutoCheckEnabled = true
 	}
-	if conf.Update.TrayAutoCheckInterval == "" {
-		conf.Update.TrayAutoCheckInterval = "24h"
+	if i := conf.Update.TrayAutoCheckInterval; i == "" || i == "24h" {
```

Comment:
same

Reply:
I thought 24h is a little too rare, for instance if we have important bugfix release. It can probably be reduced even more, checking for releases every couple of hours won't hurt anyone. What do you say?

Reply:
hm, it's ok, but it could be little bit annoying if you ignore update message and prefer to use old version

Reply:
so, if you want, it could be changed to 2 hours